client/src/components/YelpRequest.js

```diff
@@ -0,0 +1,6 @@
+import React, { useState } from "react"
+
+const YelpRequest = (props) => {
+  const [yelpRequest, setYelpRequest] = useState 
+} 
```

Comment:
haha well dont merge this in as is!

Reply:
I delete file, since the branch I branched from this branch solves this problem and I don't want to hurt my emotionally delicate main branch with sad code